doc/rules/semicolon/space_after_semicolon.rst

```diff
@@ -29,13 +29,13 @@ Example #1
    --- Original
    +++ New
     <?php
-   -                        sample();     $test = 1;
-   -                        sample();$test = 2;
-   -                        for ( ;;++$sample) {
-   +                        sample(); $test = 1;
-   +                        sample(); $test = 2;
-   +                        for ( ; ; ++$sample) {
-                            }
+   -sample();     $test = 1;
+   -sample();$test = 2;
+   -for ( ;;++$sample) {
+   +sample(); $test = 1;
+   +sample(); $test = 2;
+   +for ( ; ; ++$sample) {
+    }
```

Comment:
You can see that the string content  of the samples didn't changed because the generated doc files didn't changed.
Only exception was this sample here where I changed the indentation intentionally..